OracleManagementAgent/README.md

```diff
@@ -0,0 +1,136 @@
+<!--
+    Copyright (c) 2022, Oracle and/or its affiliates.
+    Licensed under the Universal Permissive License v 1.0 as shown at
+    https://oss.oracle.com/licenses/upl.
+-->
```

Comment:
Documentation files do not need copyright headers, so please remove this block.

Reply:
Fixed